.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,135 @@
+name: PR Review Workflow for Breaking Changes
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - labeled
+      - unlabeled
+      - synchronize
+  pull_request_review:
+    types:
+      - submitted
+      - edited
+      - dismissed
+
+jobs:
+  check_breaking_change:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for breaking change label
+        id: check_label
+        uses: actions/github-script@v6
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const { data: labels } = await github.rest.issues.listLabelsOnIssue({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+            });
+            const hasBreakingChange = labels.some(label => label.name === 'breaking change');
+            console.log(`Has breaking change label: ${hasBreakingChange}`);
+            return hasBreakingChange;
+
+      - name: Set QA reviewers
+        if: steps.check_label.outputs.result == 'true'
+        id: set_reviewers
+        run: |
+          echo "mobile_qa=churik,yevh-berdnyk,VolodLytvynenko,pavloburykh,mariia-skrypnyk,Horupa-Olena" >> $GITHUB_OUTPUT
+          echo "desktop_qa=anastasiyaig,virginiabalducci,glitchminer,antdanchenko" >> $GITHUB_OUTPUT
```

Comment:
For the time being I'm hardcoding desktop and mobile QA members here.
To fetch QA Teams from status-im github ORG I'll need another token that has ORG level permissions and that can also be subject to rate limiting. 
so for simplicity sake this was easier.

Reply:
@igor-sirotin : we can also write some logic to randomise who gets requested for review since we now decide who to request a review ;)

Reply:
This works ofc, if it's simpler 👍 We can improve in future.

Though I'm worried about pinging all QAs, we end up with the list like this:
<img width="329" alt="image" src="https://github.com/user-attachments/assets/44f47a91-b8d6-447a-8a2d-eb19e49ce1b3">

Imagine what will happen when we add devs reviews as well 🥲 
So I think the "randomise" logic is required before merging this. Otherwise we'll die in tons of notifications.

Reply:
Then can't we just put that in a separate file?

Reply:
separate file indeed looks a bit easier to manage.